themes/root/templates/search/backtoorigin.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+/** @var \Laminas\View\Renderer\PhpRenderer $this */
+/** @var \VuFind\Search\SearchOrigin\AbstractSearchOrigin|null $searchOrigin */
+if (isset($searchOrigin)) {
+    $link = $this->url('alphabrowse-home') . '?' . http_build_query($searchOrigin->getOriginUrlParamsArray());
```

Comment:
Does it make sense to hard-code the route here? Should the origin object carry the route instead for greater future flexibility?